front-end/src/api/leaderboards.ts

```diff
@@ -0,0 +1,17 @@
+import {PlayerChampionStatsRequest} from "../../../Common/Interface/Internal/player";
+import {GameRoles} from "../../../Common/Interface/General/gameEnums";
+import {MakeBackendCall} from "./_call";
+import {GetLeaderboardResponse} from "../../../Common/Interface/Internal/leaderboard";
+import PlayerStatModel from "../../../Common/models/playerstat.model";
+import {combine} from "../../../Common/utils";
+
+export async function getFlattenedLeaderboard(seasonId?: number, risenOnly?: boolean, roleId?: string) : Promise<PlayerStatModel[]> {
+    console.log(`getting leaderboard for SID: ${seasonId} RISEN: ${risenOnly} ROLE: ${roleId}`)
```

Comment:
When you refactor replace this with the logger

Reply:
We dont have a logger object in the frontend package>